src/main/scala/viper/server/frontends/http/ViperHttpServer.scala

```diff
@@ -20,28 +20,27 @@ import viper.server.ViperConfig
 import viper.server.core.{AstConstructionFailureException, VerificationExecutionContext, ViperBackendConfig, ViperCache, ViperCoreServer}
 import viper.server.frontends.http.jsonWriters.ViperIDEProtocol.{AlloyGenerationRequestComplete, AlloyGenerationRequestReject, CacheFlushAccept, CacheFlushReject, JobDiscardAccept, JobDiscardReject, ServerStopConfirmed, VerificationRequestAccept, VerificationRequestReject}
 import viper.server.utility.Helpers.{getArgListFromArgString, validateViperFile}
-import viper.server.utility.ibm
 import viper.server.vsi.Requests.CacheResetRequest
 import viper.server.vsi._
 import viper.silver.logger.ViperLogger
 import viper.silver.reporter.Message
 
+import scala.concurrent.Future
 import scala.util.{Failure, Success, Try}
 
 class ViperHttpServer(config: ViperConfig)(executor: VerificationExecutionContext)
   extends ViperCoreServer(config)(executor) with VerificationServerHttp {
 
-  override def start(): Unit = {
+  override def start(): Future[Unit] = {
```

Comment:
Same comment: Akka provides a special type ```Done``` for writing ```Future[Done]``` which IMO has a clearer intention.